px-datetime-picker.html

```diff
@@ -276,6 +346,70 @@
       if(evt.detail.dateOrTime === "Date") {
         this.$.dropdown.toggle();
       }
+    },
+    /**
+     */
+    _collapseQueryIsValid: function(query) {
+      if (typeof query === 'number') {
+        return true;
+      }
+      if (typeof query === 'string' && query.slice(-2) === 'px' && !isNaN(parseInt(query))) {
+        return true;
+      }
+      return false;
+    },
+    _getCollapseQuery: function(collapseAt) {
+      if (typeof collapseAt === 'number') {
+        return collapseAt + 'px';
+      }
+      if (typeof collapseAt === 'string' && collapseAt.slice(-2) === 'px' || !isNaN(parseInt(collapseAt))) {
+        return collapseAt;
+      }
+    },
+    /**
+     * Determine if should field be displayed in container
+     * The dt-field will only appear in the container if fillContainer is true
+     */
+    _handleField: function(hideTime, fillContainer){
+      if(!hideTime && fillContainer){
+        return true;
+      }
+      return false;
+    },
+
+    mobileKeyboard(e){
+      console.log("ran")
+    },
+
+    // ********************************************************************** //
+    // Calculate iron-dropdown properties
+    // ********************************************************************** //
+
+    _getVerticalOffset: function(fillContainer) {
+      if(!fillContainer){
+        return 20;
+      }
+      return 0;
+    },
+    _getPositionTarget: function(fillContainer, fitIntoElement) {
+      if (fillContainer){
+        return fitIntoElement;
+      } else {
+        return this.$.field;
+      }
+    },
+    _handleBelowCollapseSizeChanged: function(newVal, oldVal) {
+      if (newVal && !this.opened) {
+        this.opened = true;
```

Comment:
Let's discuss, I'm not sure I understand the use case here. I could see where you would want to transition from a panel to a fullscreen/fillContainer at a certain breakpoint, but I don't understand why you're automatically popping it open at a certain breakpoint.